conformance/tests/tlsroute-simple-same-namespace.go

```diff
@@ -53,6 +54,11 @@ var TLSRouteSimpleSameNamespace = suite.ConformanceTest{
 			t.Fatalf("unexpected error in test configuration, found %d hostnames", len(hostnames))
 		}
 		serverStr := string(hostnames[0])
+		kubernetes.HTTPRouteMustHaveCondition(t, suite.Client, suite.TimeoutConfig, routeNN, gwNN, metav1.Condition{
+			Type:   string(v1beta1.RouteConditionResolvedRefs),
+			Status: metav1.ConditionTrue,
+			Reason: string(v1beta1.RouteReasonResolvedRefs),
+		})
```

Comment:
With this PR this check suddenly becomes very common, and is now repeated in several places with the same `Type`, `Status`, and `Reason`. Given that, have you considered making a wrapper to capture the specific state we're checking for? e.g. `HTTPRouteMustHaveResolvedRefsConditionsTrue()` or something like that? :thinking:

Reply:
Yep, it makes total sense. I'll introduce an helper for that 👍